src/sentry/static/sentry/app/views/settings/project/projectOwnership/ruleBuilder.jsx

```diff
@@ -58,37 +59,20 @@ const Owners = styled('div')`
 
 const BuilderDropdownButton = styled(DropdownButton)`
   margin-right: 5px;
+  padding-right: 5px !important;
   flex: 1;
   white-space: nowrap;
   height: 37px;
-  .button-label {
-    font-size: 14px;
-    padding: 4px 8px;
-    padding-left 4px;
-  }
 `;
 
 const RuleAddButton = styled(Button)`
   width: 37px;
   height: 37px;
   flex-shrink: 0;
-
-  display: flex;
-  justify-content: center;
-
-  .button-label {
-    padding: 0.5em;
-  }
-
-  div {
-    margin: 0px !important;
-  }
+  padding: 10px 12px !important;
```

Comment:
relying less on the nested selectors makes it play nice with css in production

Reply:
😬 - We should refactor the button component